### ISSUE TITLE
feat(CLI): More details for `envd envs describe` command

### ISSUE THREAD

**Issue by VoVAllen** [tensorchord/envd#551]
## Description

Currently `envd envs describe` only outputs the dependencies. We should put more information here.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

**Comment by gaocegege** [comment]
```
DEPENDENCIES    TYPE  
via             Python


CONTAINER PORT  PROTOCOL        HOST IP         HOST PORT
2222            tcp             127.0.0.1       37439    
8888            tcp             127.0.0.1       38353    
```

We should show the service name in the ports list. Maybe like this:

```
NAME   CONTAINER PORT  PROTOCOL        HOST IP         HOST PORT
sshd     2222            tcp             127.0.0.1       37439    
jupyter  8888            tcp             127.0.0.1       38353    
```

**Comment by AlexXi19** [comment]
Interested in this issue and taking a look right now!

**Comment by gaocegege** [comment]
@AlexXi19 Thanks!

We are here to help.

**Comment by kemingy** [comment]
closed by #1201
